core/server/common/src/test/java/alluxio/master/journal/raft/RaftJournalTest.java

```diff
@@ -410,64 +408,52 @@ private void promoteFollower() throws Exception {
 
   @Test
   public void gainPrimacyDuringCatchup() throws Exception {
-    // TODO(feng): remove this test when remote journal write is deprecated
-    after();
-    try (Closeable r = new ConfigurationRule(
-        PropertyKey.MASTER_EMBEDDED_JOURNAL_WRITE_REMOTE_ENABLED, "true",
-        ServerConfiguration.global()).toResource()) {
-      before();
```

Comment:
Do we not need this config value any more?

Reply:
I don't think so. The old logic that was protected with this property was not right anyway.
Created this to reconsider this property and its client: https://github.com/Alluxio/alluxio/issues/14198